samples/04_gis_analysts_data_scientists/Training a Deep Learning model, for using it in 3D basemap Solutions..ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>In this step, <code>.las</code> files are converted to 'HDF5 binary data format'. The function export_point_dataset( )is used for this. </p><p> </p><p>Apart from that, the folder structure for the <code>data_path</code> argument must contain two folders, 'train' &amp; 'val', which should have the training and validation files, respectively. <em>(as shown in figure 1)</em></p><p> </p><p>The <code>export_point_dataset( )</code> function only needs an 'input folder' path and an 'output folder' path. The rest of the arguments have an appropriate default value, which should be valid in most cases. Here, 'extra_features' is used to include additional attributes from LiDAR data, like intensity, number of returns, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
corrected accordingly, Thanks.

Reply:
corrected accordingly, Thanks.